.gitignore

```diff
@@ -11,3 +11,7 @@ website/build/
 website/yarn.lock
 website/node_modules
 website/i18n/*
+
+.yalc
```

Comment:
Can you add details about yalc dependency - I had a brief look at https://github.com/whitecolor/yalc, and I assume this has to be installed globally according to their instructions, so we should document it in the README. Any way we could make it a hard dependency instead of installing globally (to skip the extra manual step when setting up the demo app)?

Reply:
I'm not sure yet if this should be included as a hard dependency, it was just a tool I was testing out to get something working. But I'd prefer to discuss with more people before deciding whether we should use it or a different approach.

Reply:
Also, I realized I accidentally checked in changes to package.json that use yalc... I'll revert that.